region/lowernorfair/east.json

```diff
@@ -9538,33 +9760,58 @@
                       ]
                     }
                   ],
-                  "devNote": "FIXME The Kihunters could be used as frozen platforms."
+                  "note": [
+                    "Shoot them from below.",
+                    "It is possible for the KiHunters to fall through the gaps in the side, but is unlikely.",
+                    "Jumping may lower the camera position making it harder to shoot the KiHunters.",
+                    "But once it is lowered, jump to make them active and vulnerable to damage."
+                  ]
                 },
                 {
                   "name": "Screw Kill",
                   "notable": false,
                   "requires": [
                     "h_canNavigateHeatRooms",
+                    "ScrewAttack",
                     {"or":[
                       "h_canCrouchJumpDownGrab",
```

Comment:
`h_canCrouchJumpDownGrab` doesnt need anything else? seems odd that it and hijump are easier than the wall jump

Reply:
I must have done walljump really poorly, and not tested CJDG.
In randomizers I more often than not get hit walljumping up because of how you lose invincibility.  But that's more of a me problem.